README.rst

```diff
@@ -50,7 +50,7 @@ Installation
     $ pip install django-q
 
 
--  Add `django_q` to your `INSTALLED_APPS` in your projects `settings.py`::
+-  Add `django_q` to your `INSTALLED_APPS` in your projects `settings.py` after all other apps::
```

Comment:
Why this change?
Nothing indicates that this is true.

I have this running in production

```
INSTALLED_APPS = (
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.humanize',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    'widget_tweaks',
    'model_mommy',
    'django_otp',
    'django_otp.plugins.otp_totp',
    'django_otp.plugins.otp_static',

    'main_app',  # Needs to be above django_q to overwride the qcluster config

    'django_q',
    'mailqueue',
    'qrcode',

    'custom_apps',
    'custom_apps',
    'custom_apps',
    'utils'
)
```